Sources/Interaction/Widgets/ResliceCursor/ResliceCursorLineRepresentation/index.js

```diff
@@ -325,48 +325,15 @@ function vtkResliceCursorLineRepresentation(publicAPI, model) {
     const normalPlane = resliceCursor.getPlane(resliceCursorPlaneId);
     const aboutAxis = normalPlane.getNormal();
 
-    const rotatedVector = [];
-    publicAPI.rotateVectorAboutVector(
+    const rotatedVector = vtkMaths.rotateVector(
```

Comment:
vtkMaths -> vtkMath

please run the ResliceCursor example to make sure it still work